tex/pgfcirctripoles.tex

```diff
@@ -136,421 +136,360 @@
 
 
 
-\long\def\pgfcircdeclarelogicport#1#2{
-	\pgfdeclareshape{american #1 port}
-	{
-	  \savedanchor\northwest{%
-		\pgf@y=\pgfkeysvalueof{/tikz/circuitikz/tripoles/american #1 port/height}\pgf@circ@Rlen
-		\pgf@y=.5\pgf@y
-		\pgf@x=-\pgfkeysvalueof{/tikz/circuitikz/tripoles/american #1 port/width}\pgf@circ@Rlen
-		\pgf@x=.5\pgf@x
-	  }
-	  \savedanchor\left{%
-	  	\pgf@y=0pt
-	  }
-	  \savedanchor\inOne{%
-		\pgf@y=\pgfkeysvalueof{/tikz/circuitikz/tripoles/american #1 port/height}\pgf@circ@Rlen
-		\pgf@y=.5\pgf@y
-		\pgf@y=\pgfkeysvalueof{/tikz/circuitikz/tripoles/american #1 port/input height}\pgf@y
-		\pgf@x=-\pgfkeysvalueof{/tikz/circuitikz/tripoles/american #1 port/width}\pgf@circ@Rlen
-		\pgf@x=.5\pgf@x
-	  }	  
-	  \anchor{in 1}{
-		\inOne
-	  }
-	  \anchor{in}{
-		\inOne
-	  }	  
-	  \anchor{in 2}{
-		\inOne
-		\pgf@y=-\pgf@y
-	  }
-	  \anchor{out}{
-		\northwest
-		\pgf@y=0pt
-		\pgf@x=-\pgf@x
-	  }
-  	  \anchor{center}{
-		\northwest
-		\pgf@y=0pt
-		\pgf@x=-.8\pgf@x  
-	  }
-	  \anchor{east}{
-	  	\left
-	  	\pgf@x=-.8\pgf@x  
-	  }
-	  \anchor{west}{
-	  	\left
-	  }
-	  \anchor{south}{
-		\northwest
-		\pgf@x=0pt
-		\pgf@y=-\pgf@y
-	  }
-	  \anchor{north}{
-		\northwest
-		\pgf@x=0pt
-	  }
-	  \anchor{south west}{
-		\northwest
-		\pgf@y=-\pgf@y
-	  }
-	  \anchor{north east}{
-		\northwest
-		\pgf@x=-\pgf@x
-	  }
-	  \anchor{north west}{
-		\northwest
-	  }
-	  \anchor{south east}{
-		\northwest
-		\pgf@x=-\pgf@x
-		\pgf@y=-\pgf@y
-	  }	  
-	  \backgroundpath{			
-			\pgfsetcolor{\pgfkeysvalueof{/tikz/circuitikz/color}}	
-			
-			
-			\northwest
-			\pgf@circ@res@up = \pgf@y 
-			\pgf@circ@res@down = -\pgf@y
-			\pgf@circ@res@right = -\pgf@x
-			\pgf@circ@res@left = \pgf@x
-			
-			#2
-	  
-	  }
-	}
-}
 
+%%% added 
+
+\newcount{\pgf@circ@res@count}% reserve global register
```

Comment:
At TeX level there are usually no braces around a `\newcount` (this is different from `\newcounter`, see e.g. https://www.texdev.net/2009/11/17/tex-counts-and-latex-counters/)

```suggestion
\newcount\pgf@circ@res@count% reserve global register
```